docs-ref-conceptual/authenticate-azure-cli-preview.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Use the subscription selector when logging into Azure.
+description: Learn how to select a subscription from the new subscription selector available with the `az login` command.
+author: dbradish-microsoft
+ms.author: dbradish
+ms.date: 04/09/2024
+ms.topic: conceptual
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli
+keywords: az login, authentication types, authentication methods, azure, cli login, az login powershell, cli login, sign in, azure cli  
+---
+
+# Choose your subscription when logging into Azure using Azure CLI
+
+With the preview release of Azure CLI [2.59.0](./release-notes-azure-cli.md#april-02-2024), there's a new login feature that provides a list of available tenants and subscriptions. Choose to keep the subscription and tenant that is selected by default, or type the line number of the subscription you want to use. Here are the details:
```

Comment:
Maybe you could include that the main benefit of having the list of available tenants and subscriptions is that users dont have to manually set their subscription anymore.

Reply:
@daphnemamsft , good idea. done.